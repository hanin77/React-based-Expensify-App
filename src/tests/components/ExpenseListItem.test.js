import React from 'react';
import ExpenseListItem from '../../components/ExpenseListItem';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expenses';

test('should render Expense Item', () => {
    const wrapper = shallow(<ExpenseListItem expenses={expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();

});


