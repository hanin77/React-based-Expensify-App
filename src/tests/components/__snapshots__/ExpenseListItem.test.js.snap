// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Item 1`] = `
<div>
  <Link
    to="/edit/undefined"
  >
    <h3 />
  </Link>
  <p>
    $0.00
     -
    Jan 1st, 1970
  </p>
</div>
`;
